File: app/src/main/java/nl/koenhabets/yahtzeescore/ScoreComparatorDate.java
package nl.koenhabets.yahtzeescore;

import java.util.Comparator;

public class ScoreComparatorDate implements Comparator<ScoreItem> {
    @Override
    public int compare(ScoreItem scoreItem, ScoreItem scoreItem1) {

        return scoreItem1.getDate().compareTo(scoreItem.getDate());
    }
}

File: app/src/main/java/nl/koenhabets/yahtzeescore/ScoreComparatorDate.kt
package nl.koenhabets.yahtzeescore

import java.util.*

class ScoreComparatorDate : Comparator<ScoreItem> {
    override fun compare(scoreItem: ScoreItem, scoreItem1: ScoreItem): Int {
        return scoreItem1.date.compareTo(scoreItem.date)
    }
}